recipes/tensora/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "tensora" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tensora-{{ version }}.tar.gz
+  sha256: f40787e36d6cf07d78a7644e06f18f7288d603f4279d2e014dded966fb62de3b
+
+build:
+  skip: True  # [win or py<310]
```

Comment:
Why are you skipping Windows? Note that it is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
In that case you can try: https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies